src/app/src/test/kotlin/org/imperial/mrc/hint/integration/DownloadTests.kt

```diff
@@ -92,7 +92,6 @@ class DownloadTests : SecureIntegrationTests()
         assertThat(contentLength).isGreaterThan(0)
         val bodyLength = response.body?.count()
         assertThat(contentLength).isEqualTo(bodyLength)
-        assertThat(headers["Connection"]?.first()).isNotEqualTo("keep-alive")
```

Comment:
This check was added here: https://github.com/mrc-ide/hint/pull/338/files but it's unclear to me why. Looks like the old tomcat version was not adding a `keep-alive` instruction; the new one is.